packages/apollo-server-codegen/src/IR/Descriptions.ts

```diff
@@ -0,0 +1,13 @@
+import { Translatable, Translator } from "../Translators";
+
+export class Description implements Translatable {
+  public description: string;
+  constructor(node: { description?: { value: string } }) {
```

Comment:
Is this an ASTNode? Can we use actual ASTNode typings here?

Reply:
It's not an ASTNode, as not all nodes can have descriptions (extension definition nodes, name nodes, etc)